dev-qt/qtserialbus/qtserialbus-5.9.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="Support for CAN and potentially other serial buses."
```

Comment:
I think modbus is also supported?

delete "potentially" and don't end with a period